modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -451,6 +451,9 @@ enum ContourApproximationModes {
     CHAIN_APPROX_TC89_KCOS = 4
 };
 
+CV_EXPORTS
+std::vector<Point> approximateChainTC89(std::vector<schar> chain_, Point origin, const int method);
```

Comment:
Why not CV_EXPORTS_W? What about documentation and reference to paper?

Reply:
I exported this function to be able to test it (though I haven't make such tests yet), I'm not sure whether is it useful. It transforms chain code to vector of points using one of methods, but there are no other interfaces with chain codes. New findContours can return chain code, but without origin point it is not very useful.

So export of this function is still in question. I wanted to get other opinions. Should we complete chain code output in `findContours` and expose `approximateChain` or hide these functionality?